nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/main/java/org/apache/nifi/avro/AvroRecordReader.java

```diff
@@ -24,6 +24,8 @@
 import org.apache.nifi.serialization.record.Record;
 import org.apache.nifi.serialization.record.RecordSchema;
 
+import avro.shaded.com.google.common.base.Throwables;
```

Comment:
The fully-qualified class name looks shady (pardon the pun). Is that a stable, usable, and (most importantly) necessary class?

Reply:
You are right, i didn't pay attention to the imports :-/

Now I am bringing in the official / latest of guava. 

I am using it to construct a user friendly message, that would end up in the logs / bulletin, eg.:
<img width="1597" alt="screen shot 2019-02-01 at 10 02 16" src="https://user-images.githubusercontent.com/1477846/52113107-b3399500-2608-11e9-92de-f602204cdc99.png">
without it, it's rather cryptic why the parse failed, so it is not crucial, but rather useful.